changelog/1327.doc.rst

```diff
@@ -0,0 +1,2 @@
+Updates deprecated meeting links. Deletes deprecated calendar
+ links. TO DO: update invitation links.
```

Comment:
```suggestion
Updated deprecated meeting and calendar links in :file:`README.md`.
```
A minor suggested change only to match the style of the other changelog entries.